src/h.js

```diff
@@ -1,22 +1,29 @@
-export function h(tag, data) {
-  var node
-  var stack = []
-  var children = []
+function addChild(children, val) {
+  if (val == null || typeof val === 'boolean') return
 
-  for (var i = arguments.length; i-- > 2; ) {
-    stack[stack.length] = arguments[i]
+  if (Array.isArray(val)) {
+    for (var j = 0; j < val.length; j++) {
```

Comment:
Nothing official, hence the quotation marks. Have mentioned it briefly to a few perf addicts on twitter & they're all aware of it too.

Here's a [simple gist](https://gist.github.com/lukeed/2629834a2d91a85885678ab561b3a202) of noticeable impact. Even with only 1000-item (empty) array, there's a 200ms difference.